mozilla-lint-rules/build.gradle

```diff
@@ -9,9 +9,12 @@ targetCompatibility = JavaVersion.VERSION_1_8
 sourceCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
-    compileOnly "org.jetbrains.kotlin:kotlin-stdlib:1.3.61"
-    compileOnly "com.android.tools.lint:lint-api:26.6.1"
-    compileOnly "com.android.tools.lint:lint-checks:26.6.1"
+    compileOnly Deps.kotlin_stdlib
+    compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Versions.kotlin}"
+    compileOnly Deps.kotlin_reflect
+
+    compileOnly "com.android.tools.lint:lint-api:27.0.1"
```

Comment:
The lint API version is linked to the android gradle plugin versions. If we use your PR instead of mine, we should comment to that affect and make sure the versions are aligned. See https://github.com/mozilla-mobile/fenix/pull/15646/commits/b34e654f8dd688e38d35b74f26b2f6ad61d7a69c#diff-f6dcd12435c476b2cedd79bcd292b929R9

Reply:
This is the combination we are using in AC for our custom lint checks.